restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/AsrSignal.java

```diff
@@ -46,9 +46,12 @@
     private final long timeAfterSpeech;
     private final String hotWords;
     private final String lang;
+    private final String input;
+    private final int minNumber;
+    private final int maxNumber;
 
     public AsrSignal(String driver, String lang, List<URI> initialPrompts, String endInputKey, long maximumRecTimer, long waitingInputTimer,
-                     long timeAfterSpeech, String hotWords) {
+                     long timeAfterSpeech, String hotWords, String input, int numberOfDigits) {
```

Comment:
imo this constructor deserves to have a java doc, with a brief description of these parameters.. you can take from specification document.